NEWS.md

```diff
@@ -0,0 +1,22 @@
+# sircovid 0.6.5
+
+* Parameter `N_age` was split into `n_age` (both models) and `n_groups` (carehomes)
+* Implement preliminary support for waning immunity
+
+# sircovid 0.6.4
+
+* Implement preliminary support for seroreversion
+
+# sircovid 0.6.3
+
+* Implement preliminary support for vaccination modelling
+
+# sircovid 0.6.2
+
+* Support for probability of positive calculation directly in the package
+* Pillar 2 data is now fitted to over 25s only
```

Comment:
Maybe reword this as we can still fit Pillar 2 data to all ages. Over 25s only is a new, alternative option.

Reply:
thanks!